Styles/riveraStyle.css

```diff
@@ -0,0 +1,282 @@
+
+  * {
+    font-family: 'Source Sans Pro', sans-serif;
+  }
+
+
+  .companyName {
+      font-size: 32px;
+      font-weight: 900;
+      /* margin-top: 15px; */
+      width: 240px;
+      /* height: 81px; */
+  }
+
+  /* .navbar {
+      height: 60px;
+  } */
+
+  .no-margin {
+  margin: 0px !important;
+}
+
+.firstDiv {
+    background-color: black;
+    height: 35px;
+}
+
+.navbar {
+    padding-top: 5px;
+    padding-bottom: 5px;
+}
+
+.nav-link {
+    padding-left: 2px;
+}
+
+.navbar-brand {
+    font-size: 30px;
+    font-weight: bolder;
+}
+
+/* this position relative then text is position absolute to pic */
+.carousel-item {
+    position: relative;
+    text-align: center;
+    color: white;   
+}
+
+.carousel-control-prev-icon {
+    background-color: #ED564C;
+}
+
+.carousel-control-next-icon {
+    background-color: #ED564C;
+}
+
+.centered {
+    color: white;  
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.centered > h3 {
+    font-size: 30px;
+    text-transform: uppercase;
+    font-weight: 100;
+    color: #dadcdd;
+    opacity: 0.7;
+}
+
+.centered > p {
+    font-size: 12px;
+    color: #dadcdd;
+    opacity: 0.5;
+}
+
+/* turn imagees black and white */
+.d-block  {
+    -webkit-filter: grayscale(80%); /* Safari 6.0 - 9.0 */
+    filter: grayscale(100%);
+}
+
+/* .doverlay {
```

Comment:
Will this commented code still be used at some point?
Same question goes for other commented code.